Create Post Button

alaasaedy opened the pull request:
- Add Create Post Button To Homepage

- Add router to direct users forward New Post page

alaasaedy committed:
- Add Create Post Button  To Homepage

alaasaedy committed:
Add router to button to direct user to New Post route

alaasaedy committed:
- Seprate changes in CreatePostButton component and add it to homepage.

MAlngaawy commented on src/components/CreatePostButon/index.jsx:
```diff
@@ -0,0 +1,39 @@
+          color={BUTTON_OPTIONS.BACKGROUND_COLOR.PrimaryBlack}
+          className="flex flex-shrink py-3 px-4 my-6"
+        >
+          <i className="mr-3 hidden sm:block h-8 w-8">
```
- don't hide the Icon on the small screen
- the clickable area should be just the button, not the whole page div
- make the icon a little bit small, give it a (w-5 h-5) classes instead of 8
- and it would be better if you can find an icon with a grey color or less black

MAlngaawy reviewed (changes requested):
nice work Alaa, but consider these changes, please

alaasaedy committed:
- Fix Button clicable area
- decrease icon inside img dimensions
- remove unnecessary hide class
- less black color icon

alaasaedy committed:
- increase button px

MAlngaawy committed:
handle some styles
